integtest/spec/single_book_spec.rb

```diff
@@ -440,12 +440,39 @@ def self.index
       @opened_docs.exit
     end
 
+    let(:static) { 'http://localhost:8000/guide/static' }
     let(:index) { Net::HTTP.get_response(URI('http://localhost:8000/guide/')) }
+    let(:js) do
+      Net::HTTP.get_response(URI("#{static}/docs.js"))
+    end
+    let(:css) do
+      Net::HTTP.get_response(URI("#{static}/styles.css"))
+    end
+
     it 'serves the book' do
       expect(index).to serve(doc_body(include(<<~HTML.strip)))
         <a href="chapter.html">Chapter
       HTML
     end
+    context 'the js' do
+      it 'is unminified' do
+        expect(js).to serve(include(<<~JS))
```

Comment:
These tests are a bit brittle, but ok

Reply:
Yeah. I thought they weren't great. Do you have any recommendations?

Reply:
I don't really, other than maybe we don't need to test for it or we put some text to look for in index.js and comment it with `// used in an assertion, don't remove`

Reply:
I'll use a comment like that, yeah.